src/index.ts

```diff
@@ -883,6 +883,19 @@ export class Bigtable {
     }
   }
 
+  /**
+   * Terminate grpc channels and close all the clients.
+   */
+  close(): Promise<void[]> {
```

Comment:
any good way to test this?

Reply:
Maybe we can add a test to make sure request will fail after client is closed, I'll work on it next week!